### ISSUE TITLE
`make down/basic` does not stop morph

### ISSUE THREAD

**Issue by carpawell** [nspcc-dev/neofs-dev-env#224]
After #212 there is a new target in the `Makefile`: `(up|down)/bootstrap`. It has been added to every `up/*` but not to all the `down/*` targets.
As a result `down/basic` does not return a state after the `up/basic`. It confused me a little and was not expected of me: every local run was made using the same morph state without any cleaning. Not sure if it is something that we want to have in our makefile. Mb it is a kinda feature and allows caching morph b/w runs.

**Comment by cthulhu-rider** [comment]
i find this behavior correct:
1. `make down/basic` stops services listed in `.basic_services` and no others
2. if `make up/bootstrap` was done 1st, then `make up/basic`, i don't expected `make down/basic` to stop the bootstrap that were run 1st

P.S. after #282 `ir` (ex. `morph_chain`) keeps the NeoFS chain between starts intentionally
- wanna stop all services, do `make down`
- wanna reset collected state, do `make clean`  
- and if u don't, this does not happen

**Comment by carpawell** [comment]
@cthulhu-rider, dont you think this scenario should be correct:

1. i have nothing
2. i do `make up/basic`
3. i do `make down/basic`
4. i have nothing again?

Why are they named the opposite way then if `down` does not neutralize `up`? Isnt `morph` basic too?

> make down/basic stops services listed in .basic_services and no others

Agree, but files/services grouping is kinda of an internal thing. What can `make up/bootstrap` be useful for? Running `neo-go` in the `neofs-devenv` repo?
I think we can reconsider devenv (and this issue) soon after it is not used in the integrational tests (mb that will also mean #212 rollback).

**Comment by roman-khimov** [comment]
Might become irrelevant after #282.

**Comment by cthulhu-rider** [comment]
@carpawell if paraphrase the original issue text to `lets embed "bootstrap" services into the "basic" ones and purge "bootstrap" as a separate component`  - im only :+1:

**Comment by carpawell** [comment]
> lets embed "bootstrap" services into the "basic" ones and purge "bootstrap" as a separate component

Yes, i think it is the closest one to what i would implement.

**Comment by roman-khimov** [comment]
After some poking around (including #347) I'd suggest closing it as WONTFIX:
 * storage nodes need GAS and it has to be managed in-between IR and SN startup, so a flat basic service list can't work
 * this means we still need a list of "bootstrap" services in some way
 * we can drop `*/bootstrap` targets while keeping the list, but that doesn't seem to solve any problem
 * we have a similar behavior for `down/add`, there can be a case when you'd like to stop a subset of services
 * we have this behavior for four years now
